cmd/rootCmd.go

```diff
@@ -58,14 +61,27 @@ func NewCmdRoot(outWriter, errWriter io.Writer) *cobra.Command {
 	})
 
 	CheckForUpdate(configuration)
+	rootCmd.AddGroup(
+		&cobra.Group{
+			ID:    "deployment",
+			Title: "Deployment Commands:",
+		},
+		&cobra.Group{
+			ID:    "management",
+			Title: "Management Commands:",
```

Comment:
Thoughts on calling this "Administrative Commands"?

Reply:
good thinking, I'll call them that